releasenotes/2.1.1.md

```diff
@@ -0,0 +1,75 @@
+* [JCLOUDS-1362] - Randomly generated node password can be invalid in Azure ARM
+* [JCLOUDS-1368] - GCS cannot upload multipart file larger than 1 GB
+* [JCLOUDS-1372] - Add support for R4 instance types in AWS ec2
+* [JCLOUDS-1373] - Add the new Paris region to aws-ec2
+* [JCLOUDS-1381] - NullPointerException provisioning VM in GCE: could not find image for disk
+* [JCLOUDS-1387] - Add support for c5 instance types in AWS ec2
+* [JCLOUDS-1389] - Multipart upload to Google cloud storage failing with 400 bad request
+* [JCLOUDS-1391] - Headers must be sorted by lowercase when calculating signature
+* [JCLOUDS-1392] - When fetching blob metadata with details all folders are removed from response
+* [JCLOUDS-1400] - GCS's presigned url requires blank Content-Type
+* [JCLOUDS-1401] - S3 SignatureDoesNotMatch when listing keys using prefix with ampersand
+* [JCLOUDS-1405] - Dimension Data Live Tests Broken - Domain classes for Server and Image with List nics incorrect
+* [JCLOUDS-1410] - SoftLayer cannot provision with private image
+* [JCLOUDS-1422] - LocalBlobStore.list ignores recursive flag when prefix set
+* [JCLOUDS-1427] - NPE when deserializing Elastic IPs that have tags
+* [JCLOUDS-1433] - IllegalArgumentException from Strings2.replaceTokens
+* [JCLOUDS-1436] - Add support for m5 instance types in AWS ec2
+
+The complete list of fixed issues and improvements can be found [here](https://issues.apache.org/jira/secure/ReleaseNote.jspa?projectId=12314430&version=12342884).
+
+## <a id="knownissues"></a> Known Issues
+
+There are a number of open JIRA issues related to OpenStack endpoint and API version support: [JCLOUDS-114](https://issues.apache.org/jira/browse/JCLOUDS-114), [JCLOUDS-773](https://issues.apache.org/jira/browse/JCLOUDS-773) and [JCLOUDS-1197](https://issues.apache.org/jira/browse/JCLOUDS-1197).
+
+## <a id="credits"></a>Credits
+
+jclouds would like to thank everyone who contributed time and effort in order to make this release happen:
+
+* [Vikas Rangarajan](https://github.com/VRanga000) for his contribution to the new instance types supported in the AWS ec2 provider.
+* [Jesse Glick](https://github.com/jglick) for the fix for AWS S3 signatures.
+* [Alix Lourme](https://github.com/axel3rd) for the Openstack Keystone V3 domain configuration for projects.
+* [Daniel Estevez](https://github.com/danielestevez) for his contributions to the Azure ARM provider.
+* Apache jclouds [PMC](http://people.apache.org/committers-by-project.html#jclouds-pmc) and community for verifying the release.
+* Check out who has been busy on [Open Hub](https://www.openhub.net/p/jclouds/contributors?query=&sort=latest_commit).
+
+## <a id="test"></a>Test Results
+
+Please see the discussion and the vote threads for live test results for 2.1.1:
+
+* RC1 [vote](https://s.apache.org/jclouds211rc1vote) and [discussion](https://s.apache.org/jclouds201rc1discuss) threads.
```

Comment:
`jclouds211rc1discuss` rather than `jclouds201rc1discuss`? I assume those two shortlinks are intended not to work yet?

Reply:
Yes, it needs to be updated to 211. These are just placeholders, but since the Apache URL shortener allows you to pick a name for the generated URL, we can use that so we can build the release notes and avoid having to edit them when a RC email or the announcement email itself are sent.

Reply:
And it's done! Thanks @demobox for noticing